README.md

```diff
@@ -121,6 +121,8 @@ You can specify default option as follows.
   - `fr` french
   - `fi` finnish
   - `sv` swedish
+  - `zh_Hans_CN` Chinese Simplified
+  - `zh_Hant_CN` Chinese Traditional
```

Comment:
Can we use `-` for language code instead of `_`? I think `-` is generally used to express a language code.

Also, do we need to add `CN` in the last of code? If there are any differences in translations by regions like China, Hong Kong , Taiwan..., maybe `CN` is necessary. But we may not need it in this plugin. I think shorter code might be better

Reply:
OK